lib/plug/ssl.ex

```diff
@@ -197,6 +267,36 @@ defmodule Plug.SSL do
     |> Keyword.put_new(:reuse_sessions, true)
   end
 
+  defp configure_managed_tls(options) do
+    case Keyword.get(options, :cipher_suite) do
+      :strong -> set_strong_tls_defaults(options)
+      :compatible -> set_compatible_tls_defaults(options)
+      _ -> Keyword.delete(options, :cipher_suite)
+    end
+  end
+
+  defp set_managed_tls_defaults(options) do
+    options
+    |> Keyword.delete(:cipher_suite)
+    |> Keyword.put_new(:honor_cipher_order, true)
+    |> Keyword.put_new(:client_renegotiation, false)
```

Comment:
It his part of the cipher_suite option or should we just always set it to false regardless like we do to `renew_sessions` or `secure_renegotiate`?